packages/easy-email-localization/locales/en.json

```diff
@@ -23,5 +23,189 @@
   "Page": "Page",
   "Section": "Section",
   "Column": "Column",
-  "Text": "Text"
-}
\ No newline at end of file
+  "Text": "Text",
+  "JSON source": "JSON source",
+  "MJML source": "MJML source",
+  "Content": "Content",
+  "Layout": "Layout",
+  "2 columns": "2 columns",
+  "3 columns": "3 columns",
+  "4 columns": "4 columns",
+  "Block": "Block",
+  "Align": "Align",
+  "Background image": "Background image",
+  "Background color": "Background color",
+  "Background repeat": "Background repeat",
+  "Background size": "Background size",
+  "Border radius": "Border radius",
+  "Border": "Border",
+  "Font style": "Font style",
+  "Color": "Color",
```

Comment:
Restore this file, we extract keywords and translate by  scripts.

Reply:
I didnt understand what you meant here and below.